src/media/js/main.js

```diff
@@ -144,7 +144,9 @@ require(
 
     window.addEventListener(
         'resize',
-        _.debounce(function() {z.doc.trigger('saferesize');}, 200),
+        _.debounce(function() {
+            document.dispatchEvent(new CustomEvent('saferesize'));
```

Comment:
CustomEvent constructor might not be supported in all browsers. I'd include element_utils.MktEvent to make sure it's "polyfill"ed